src/runtime_src/core/pcie/windows/device_windows.cpp

```diff
@@ -198,6 +198,152 @@ info_mgmt(const device_type* device, qr_type qr, const std::type_info&, boost::a
   }
 }
 
+static void
+sensor_info(const device_type* device, qr_type qr, const std::type_info&, boost::any& value)
```

Comment:
First let me say, that I understand the changes you are making and how you are working on "fitting" them into the code flow.  That said, I do have an issue with the "large" switch statement.  It would be great to place all of the metadata associated with gathering this data into a "collection" and use that collection to obtain the query of choice. 

Does this work need to be done for this PR, I don't think so, but it _**should be done in the next two weeks**_.  That said, could you add a ticket to our scrum board and schedule it to be done in our next sprint.

Reply:
Ok. Sounds good. I'll add a ticket for this